src/libraries/System.Formats.Tar/src/System/Formats/Tar/TarReader.cs

```diff
@@ -227,11 +215,7 @@ private bool TryGetNextEntryHeader(out TarHeader header, bool copyData)
             Debug.Assert(!_reachedEndMarkers);
 
             header = default;
-
-            // Set the initial format that is expected to be retrieved when calling TarHeader.TryReadAttributes.
-            // If the archive format is set to unknown here, it means this is the first entry we read and the value will be changed as fields get discovered.
-            // If the archive format is initially detected as pax, then any subsequent entries detected as ustar will be assumed to be pax.
-            header._format = Format;
+            header._format = TarEntryFormat.Unknown;
```

Comment:
I believe this is a redundant assignment, and feels like it should just be overwritten by the next line anyways.

Reply:
Fixed.